src/components/modules/AverageTimeReports/types.ts

```diff
@@ -0,0 +1,30 @@
+export interface AirQuality {
```

Comment:
This can be moved to `src/types`. I'd prefer to keep all types in the `src/types` directory except component props types and some types related only to a single component.